website_and_docs/content/documentation/webdriver/bidirectional/bidi_api_remotewebdriver.zh-cn.md

```diff
@@ -1,65 +1,54 @@
 ---
-title: "RemoteWebDriver BiDirectional API"
-linkTitle: "RemoteWebDriver BiDi API"
+title: "远程网络驱动 双向 API（RemoteWebDriver BiDirectional API）"
+linkTitle: "远程网络驱动 BiDi API"
```

Comment:
Can you keep the WebDriver as a Proper Noun ？

Reply:
Yes,i can.
I will to change it.